packages/oak-addon-richtext-field-prosemirror/lib/core/Editor/Toolbar/ColorButton.js

```diff
@@ -6,13 +6,13 @@ import {
   DropdownToggle,
   Tooltip,
   classNames,
-} from '@poool/junipero';
+} from '@junipero/react';
 import { Text } from '@poool/oak';
 
 import { schema } from '../schema';
 import { getActiveAttrs } from '../utils';
 
-export default ({ className, onChange, state, active = false }) => {
+export default ({ className, onChange, state }) => {
```

Comment:
you remeber why you removed active ?
(btw if its completely useless now, we can can remove this from props on Toolbar/index.js)

Reply:
The color button never had a real active state (nor it should, not like Bold or Italic), it should only reflect the current text color anytime. It can also be safely removed from Toolbar, will do